apps/accounts/views.py

```diff
@@ -161,7 +161,7 @@ def get(self, request, user, identity, next_path):
             return HttpResponseRedirect(path)
         else:
             login_user(request, user, identity)
-            next_path = next_path or '/'
+            next_path = next_path or reverse('home')
             log.info('Redirecting after login to: {}'.format(next_path))
             return HttpResponseRedirect(next_path)
```

Comment:
I figured as much but it would be less error prone if `is_safe_url()` was called at the very last minute before the redirect.